Sources/Sentry/SentryNetworkTrackingIntegration.m

```diff
@@ -10,45 +10,49 @@ @implementation SentryNetworkTrackingIntegration
 
 - (void)installWithOptions:(SentryOptions *)options
 {
-    // We are aware that the SDK only creates breadcrumbs for HTTP requests if performance is
-    // enabled. A proper fix is not straight forward as we need several checks on SentryOptions in
-    // SentryNetworkTracker. As we have a problem with KVO, see
-    // https://github.com/getsentry/sentry-cocoa/issues/1328, we don't know if we can keep the
-    // SentryNetworkTracker (written on 29th of September 2021). Therefore we accept this tradeof
-    // for now.
+    if (!options.enableSwizzling) {
+        [SentryLog logWithMessage:
+                       @"Not going to enable NetworkTracking because enableSwizzling is disabled."
+                         andLevel:kSentryLevelDebug];
+        return;
+    }
+
+    BOOL needNetworkTracking = true;
```

Comment:
`m`: `BOOL` is `YES`. Same applies for usages below. Maybe the following name suits better.
```suggestion
    BOOL shouldEnableNetworkTracking = YES;
```